intern.json

```diff
@@ -1,4 +1,13 @@
 {
+  "description": "Default test suites",
+  "debug": true,
+  "filterErrorStack": true,
   "suites": "tests/**/*.js",
-  "environments": ["node"]
-}
\ No newline at end of file
+  "node": {
+    "plugins":
+      "node_modules/babel-register/lib/node.js"
```

Comment:
This is cool... I never really took the time to learn about this: https://github.com/theintern/intern/blob/master/docs/configuration.md#configuring-plugins